ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -81,11 +81,11 @@ fi
 
 # multi-instance setup
 # shellcheck disable=SC2154
-param set MAV_SYS_ID $((1+px4_instance))
-simulator_udp_port=$((14560+px4_instance))
-udp_offboard_port_local=$((14557+px4_instance))
-udp_offboard_port_remote=$((14540+px4_instance))
```

Comment:
14540 also needs to stay. Generally you can change local ports but not remote ones (at least for the first instance).